Cognigy/WebchatWidget
# Improve Carousel Accessibility

This PR improves the accessibility of carousels.
Success criteria:
1. Carousel that has focus should be scrollable using arrow keys. This way, the user using a keyboard can view all cards, even if a card does not have an interactive element
2. Next Button of the carousel will not appear while tabbing out of the last item in the carousel. 
3. Each Carousel Item should have proper numbering indicating its position in the carousel

To test:
- Create Say node to include a gallery. Configure the gallery to include multiple cards with buttons. Also, include a card without a button in between other cards.
- Use the keyboard to move the focus to one of the items within the carousel. Now use right/left arrows to scroll through the list of cards.
- Move focus to the last button within the last card of the carousel. Now, on clicking Tab, the Next button should not appear. 
- On navigating the carousel using a screen reader, you should hear the list item position along with the title. (Example: list item 2 of 3: Some title)
